src/common/integration/urlProcessing.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as React from 'react';
+
+import { BookingState } from '../../context/BookingState';
+
+export const selectUrlBooking = (ComponentToWrap: React.ComponentType<any>) => {
```

Comment:
I would avoid any if possible =>

`export const selectUrlBooking = <Props>(ComponentToWrap: React.ComponentType<Props>) => {`

^^ something like that should work